lib/rubygems/commands/sign_command.rb

```diff
@@ -41,7 +46,8 @@ def execute
     gemfile = Gem::Sigstore::Gemfile.new(get_one_gem_name)
     rekor_entry = Gem::Sigstore::GemSigner.new(
       gemfile: gemfile,
-      config: Gem::Sigstore::Config.read
+      config: Gem::Sigstore::Config.read,
+      token: options[:identity_token],
```

Comment:
Nitpick: perhaps `token:` should be `identity_token:` for consistency with `:identity_token` and `--identity-token`.